commands/src/main/java/org/wildfly/extras/creaper/commands/infinispan/cache/AbstractAddCache.java

```diff
@@ -0,0 +1,100 @@
+package org.wildfly.extras.creaper.commands.infinispan.cache;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+
+/**
+ * @author Jan Martiska
+ */
+public abstract class AbstractAddCache implements OnlineCommand {
```

Comment:
We don't do `@author`s in Creaper. It's visible from Git history.

Also, I don't think this class should be `public`.